Communication/Identities/README.md

```diff
@@ -0,0 +1,28 @@
+# Identities
+
+Connector identities according to this specification are typically represented by some sort of token.
```

Comment:
Connector identities comprise of a conjunction of base identity, such as X.509 certificate and some sort of token.